bugbug/models/__init__.py

```diff
@@ -29,6 +29,7 @@
     "testfailure": "bugbug.models.testfailure.TestFailureModel",
     "tracking": "bugbug.models.tracking.TrackingModel",
     "uplift": "bugbug.models.uplift.UpliftModel",
+    "bugtypeclassification": "bugbug.models.bug_type_classification.BugTypeClassificationModel",
```

Comment:
We should merge this with the pre-existing bugtype model (just overwrite bugbug/models/bugtype.py with the contents from bugbug/models/bug_type_classification.py, and then remove bugbug/models/bug_type_classification.py).